openapi/api_backend.template.yaml

```diff
@@ -1362,7 +1378,9 @@ definitions:
     description: Describe the current session of an authenticated user.
     properties:
       spidLevel:
-         $ref: '#/definitions/SpidLevel'
+        $ref: '#/definitions/SpidLevel'
+      lollipop_assertion_ref:
```

Comment:
This is the only "snake case" property here.

Reply:
Fixed in [e6da686](https://github.com/pagopa/io-backend/pull/993/commits/e6da6869899f39075d818e200d9dfe46bc9cf448)